ansible_collections/arista/avd/docs/plugins/Filter plugins/decrypt.md

```diff
@@ -0,0 +1,48 @@
+# decrypt
+
+!!! note
+    Always use the FQCN (Fully Qualified Collection Name) `arista.avd.decrypt` when using this plugin.
+
+Decrypt supported EOS passwords
+
+## Synopsis
+
+The filter is used to decrypt supported EOS passwords into clear text\.
+
+Note \- For now this filter only supports decryption from type <code>7</code> and not type <code>8a</code> for OSPF and BGP passwords\.
+
+## Parameters
```

Comment:
The escape sequence is present in the [table](https://ansible-avd--3061.org.readthedocs.build/en/3061/docs/plugins/Filter%20plugins/decrypt.html#parameters).

Reply:
I think I got rid of them now.